AWS/aws-services.json

```diff
@@ -2098,5 +2098,48 @@
       "me-south-1",
       "us-east-2"
     ]
+  },
+  {
+    "service": "ElasticLoadBalancing",
+    "resources": [
+      {
+        "resource": "LoadBalancer",
+        "resourceType": "AWS::ElasticLoadBalancing::LoadBalancer"
+      }
+    ],
+    "regions": [
+      "ap-east-1",
+      "ap-south-1",
+      "ap-south-2",
+      "ap-southeast-1",
+      "eu-north-1",
+      "eu-west-2",
+      "me-south-1",
+      "us-east-2",
+      "us-west-1",
+      "us-west-2",
```

Comment:
Could you verify region with 
```unix
 aws ssm get-parameters-by-path \
    --path /aws/service/global-infrastructure/services/elb/regions \
    --query 'Parameters[].Value'
```

Reply:
<img width="451" alt="Screenshot 2023-08-10 at 10 43 55" src="https://github.com/plerionhq/plerion-asset-counter/assets/139095828/18433e02-f5ae-451d-bf49-f426d4b7e958">

 Verified 🎉